lldb/source/Plugins/OperatingSystem/SwiftTasks/OperatingSystemSwiftTasks.cpp

```diff
@@ -180,21 +180,52 @@ FindTaskAddresses(TaskInspector &task_inspector,
   return task_addrs;
 }
 
-static std::optional<uint64_t> FindTaskId(addr_t task_addr, Process &process) {
-  size_t ptr_size = process.GetAddressByteSize();
+static llvm::SmallVector<std::optional<uint64_t>>
+FindTaskIds(llvm::ArrayRef<std::optional<addr_t>> maybe_task_addrs,
+            Process &process) {
+  llvm::SmallVector<std::optional<uint64_t>> final_results;
+  llvm::SmallVector<addr_t> to_read;
```

Comment:
If it's unlikely we hit the small case I would just use a vector here, but in leaf functions the SmallVector doesn't really hurt, so maybe you can ignore this :-)